base/doc/clsguide.tex

```diff
@@ -1567,312 +1058,255 @@ \subsection{Normalising spacing}
-please put a note in the newly created package or class file to record
-that it runs with the new standard \LaTeX{}; then distribute it to
-your users.
-
-\subsection{Try it first!}
-\label{Sec:try-it}
-
-The first thing you should do is to test your style in `compatibility
-mode'.  The only change you need to make in order to do this is,
-possibly, to change the extension of the file to |.cls|: you should
-make this change only if your file was used as a main document style.
-Now, without any other modifications, run \LaTeXe{} on a document that
-uses your file.  This assumes that you have a suitable collection of
-files that tests all the functionality provided by your style file.
-(If you haven't, now is the time to make one!)
-
-You now need to change the test document files so that they are
-\LaTeXe{} documents: see \emph{\usrguide} for details of how to do
-this and then try them again.  You have now tried the test documents
-in both \LaTeXe{} native mode and \LaTeX~2.09 compatibility mode.
-
-\subsection{Troubleshooting}
-\label{Sec:trouble}
-
-If your file does not work with \LaTeXe{}, there are two likely
-reasons.
-\begin{itemize}
-\item \LaTeX{} now has a robust, well-defined designer's interface for
-  selecting fonts, which is very different from the \LaTeX~2.09
-  internals.
-\item Your style file may have used some \LaTeX~2.09 internal commands
-  which have changed, or which have been removed.
-\end{itemize}
+An important use of this feature is to correct a problem, reported by Donald
+Arseneau, that punctuation in page headers has always (in all known \TeX{}
+formats) been potentially wrong whenever a page break happens while a local
+setting of the space codes is in effect. These space codes are changed by, for
+example, the command \verb|\frenchspacing|) and the \textsf{verbatim}
+environment.
 
-When you are debugging your file, you will probably need more
-information than is normally displayed by \LaTeXe.  This is achieved
-by resetting the counter |errorcontextlines| from its default value of
-$-1$ to a much higher value, e.g.~999.
+It is normally given the correct definition automatically in |\begin{document}|
+and so need not be explicitly set; however, if it is explicitly made non-empty
+in a class file then automatic default setting will be over-ridden.
 
-\subsection{Accommodating compatibility mode}
+\section{Deprecated commands in wide use}
 
-Sometimes an existing collection of \LaTeX~2.09 documents makes it
-inconvenient or impossible to abandon the old commands entirely.
-If this is the case, then it is possible to accommodate both conventions
-by making special provision for documents processed in compatibility
-mode.
+A small number of commands were introduced as part of \LaTeXe{} in the
+mid-1990s, are widely used but have been superseded by more modern methods.
+These are covered here as they are likely to be encountered routinely in
+existing classes and packages.
```

Comment:
IMHO, authors may also want to know what are the corresponding modern replacements.

Reply:
Also, this title sounds odd (to me):

Reply:
"... in wide use"
Better: 
"Widely used but deprecated commands"

Reply:
Please check my updated suggestion

Reply:
A suggested rewording:

There is a small number of commands, added to \LaTeXe{} in the
mid-1990s, that are widely used but have since been superseded by more modern methods.
These are described in this section because they are likely to be encountered frequently in existing classes and packages.

What happened to this section's title?

Reply:
Maybe better: 
. . . that have since been superseded by more modern methods but remain widely used.